src/Akka.Persistence.Linq2Db.Compatibility.DockerTests/SqlServer/SqlServerCompatibilitySpecConfig.cs

```diff
@@ -31,15 +31,13 @@ class = ""Akka.Persistence.SqlServer.Snapshot.SqlServerSnapshotStore, Akka.Persi
         linq2db {{
             class = ""{typeof(Linq2DbSnapshotStore).AssemblyQualifiedName}""
             plugin-dispatcher = ""akka.persistence.dispatchers.default-plugin-dispatcher""
-            #plugin-dispatcher = ""akka.actor.default-dispatcher""
             connection-string = ""{DbUtils.ConnectionString}""
-            provider-name = """ + LinqToDB.ProviderName.SqlServer2017 + $@"""
-            table-compatibility-mode = sqlserver
-            tables {{
-                snapshot {{ 
-                    auto-init = true
-                    warn-on-auto-init-fail = false
-                    table-name = ""{tableName}""    
+            provider-name = ""{LinqToDB.ProviderName.SqlServer2017}""
+            table-mapping = sql-server
```

Comment:
My biggest concern with this is that `table-mapping` is somewhat inconsistent with overall *intent* of this being a compatibility mapping for the old plugin.

i.e. if there is no `table-compatibility-mode` set, we don't use `metadata` at all for deletes or maxSeq checks (which is of course faster.)